packages/angular/cli/README.md

```diff
@@ -178,12 +181,34 @@ You can also use `ng new foo --link-cli` to automatically link the `@angular/cli
 Please read the official [npm-link documentation](https://docs.npmjs.com/cli/link)
 and the [npm-link cheatsheet](http://browsenpm.org/help#linkinganynpmpackagelocally) for more information.
 
-To run the Angular CLI test suite use the `node tests/run_e2e.js` command.
-It can also receive a filename to only run that test (e.g. `node tests/run_e2e.js tests/e2e/tests/build/dev-build.ts`).
+To run the Angular CLI test suite use the `node ./tests/legacy-cli/run_e2e` command.
```

Comment:
Can you specify here that these instructions are for the E2E suite only please?

If you have time, instructions for the unit test and large unit test suite would be nice as well (`npm test` and `npm test -- --large`).

Reply:
No problem, I'll add the missing informations soon.